src/containers/PodcastPage/PodcastSeriesPage.tsx

```diff
@@ -158,7 +157,7 @@ const PodcastSeriesPage = ({
           {podcastSeries.episodes?.length ? (
             <>
               <h2>{t('podcastPage.episodes')}</h2>
-              {podcastSeries.episodes?.map(episode => (
+              {compact(podcastSeries.episodes).map(episode => (
```

Comment:
Er det en sjanse for at vi har falsy verdier i denne?

Reply:
Fikk errors tidligere. Antakelig løst av nye gql-skjemaer